pkg/reconciler/taskrun/validate_resources.go

```diff
@@ -126,18 +123,6 @@ func missingParamsNames(neededParams []string, providedParams []string, paramSpe
 	return missingParamsNamesWithNoDefaults
 }
 
-func extraParamsNames(ctx context.Context, neededParams []string, providedParams []string) []string {
-	// If alpha features are enabled, disable checking for extra params.
-	// Extra params are needed to support
-	// https://github.com/tektoncd/community/blob/main/teps/0023-implicit-mapping.md
-	// So that parent params can be passed down to subresources (even if they
-	// are not explicitly used).
-	if config.FromContextOrDefaults(ctx).FeatureFlags.EnableAPIFields != "alpha" {
-		return list.DiffLeft(providedParams, neededParams)
-	}
-	return nil
-}
-
```

Comment:
this is not relevant to the promotion, right? i recommend that this goes in a separate commit & PR

Reply:
Leaving this in makes the tests fail. It seems like the default action was to check if there are extra parameters and throw an error. When the alpha feature flag was enabled, this would not be the case. Now that propagated params is being promoted to beta, we cannot leave this filter in just for "alpha", we need to do the same for "beta" as well. That's why I thought it belonged in the same PR.

Reply:
I'm happy to open a separate PR just for this and fix it there. However, it will have to be merged in for propagated params to beta to work.